### PR TITLE
ui: add deposit panel

### LINKED ISSUE #31: UI: add deposit panel
Both for manager and stakeholder mode:

 - [ ] Add button action "Deposit" to sidebar
 - [ ] Add DepositState get `getdepositaddress` result
 - [ ] Add DepositView: display deposit_address both with a text field with clipboard button and a QR code

### PR DESCRIPTION
**Pull request by edouardparis**
close #31

### CODE REVIEW

File: src/revaultd/model.rs [hunk 1]
```diff
@@ -1,6 +1,13 @@
 use bitcoin::{util::psbt::PartiallySignedTransaction, Transaction};
 use serde::Deserialize;
 
+/// getdepositaddress response
+#[derive(Debug, Clone, Deserialize)]
+pub struct DepositAddress {
+    #[serde(with = "bitcoin_address")]
+    pub address: bitcoin::Address,
```
Comment (darosior) [inline src/revaultd/model.rs:8]: Address does implement Deserialize already

File: src/ui/view/deposit.rs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        }
+        if let Some(qr_code) = self.qr_code.as_mut() {
+            col = col.push(Container::new(QRCode::new(qr_code).cell_size(5)));
+        }
+        if let Some(addr) = address {
+            col = col.push(Container::new(
+                Row::new()
+                    .push(Container::new(text::bold(text::small(&format!(
+                        "{}",
+                        addr
```
Comment (darosior) [inline src/ui/view/deposit.rs:55]: supernit: you could use to_string()

**Review by darosior — COMMENTED** [review]
LGTM, just a small nit wrt `Address`'s serde impl

**Comment by darosior** [comment]
Build error is:
```
error[E0061]: this function takes 2 arguments but 3 arguments were supplied
   --> src/ui/view/sidebar.rs:107:13
    |
107 |             button::primary(
    |             ^^^^^^^^^^^^^^^ expected 2 arguments
108 |                 &mut self.deposit_menu_button,
    |                 -----------------------------
109 |                 button::button_content(Some(deposit_icon()), "Deposit"),
    |                 -------------------------------------------------------
110 |                 Message::Menu(Menu::Deposit),
    |                 ---------------------------- supplied 3 arguments
    |
note: function defined here
   --> src/ui/component/button.rs:29:9
    |
7   |             state: &'a mut button::State,
    |             ----------------------------
8   |             content: Container<'a, T>,
    |             -------------------------
...
29  | button!(primary, PrimaryStyle, color::PRIMARY, color::FOREGROUND);
    |         ^^^^^^^
```

**Comment by edouardparis** [comment]
Sorry for the notif :( I pushed the fix

**Review by darosior — APPROVED** [review]
re-ACK 63f3a048e773436dec9ebd2dd1bf7c6276c556cb

**Pull request merged**